base/linalg/dense.jl

```diff
@@ -486,9 +505,22 @@ If `factorize` is called on a Hermitian positive-definite matrix, for instance,
 will return a Cholesky factorization.
 
 Example:
-```julia
-A = diagm(rand(5)) + diagm(rand(4),1); #A is really bidiagonal
-factorize(A) #factorize will check to see that A is already factorized
+```jldoctest
+julia> A = Bidiagonal(ones(5, 5), true) #A is really bidiagonal
```

Comment:
Ah, I missed the idea behind the second part of the example. I added the `Array` constructor to it.